model/error/_meta/fields.yml

```diff
@@ -17,7 +17,7 @@
 
     - name: error id icon
       type: keyword
-      script: "doc['error.grouping_key'].value"
+      script: "doc['error.grouping_key'].size() == 0 ? null : doc['error.grouping_key'].value"
```

Comment:
Not sure if `null` is the best default value. LMK if you have better suggestions.